detection_rules/main.py

```diff
@@ -47,7 +45,7 @@ def root(ctx, debug):
               help='Type of rule to create')
 def create_rule(path, config, required_only, rule_type):
```

Comment:
type hint here, if it's not supposed to be  a `Path`. or make it `Path` and fix the caller